app/src/main/java/com/android/sample/ui/profile/EditProfile.kt

```diff
@@ -210,6 +196,18 @@ fun EditProfileScreen(
 
                 Button(
                     onClick = {
```

Comment:
Added logic to delete the profile picture from the ImageViewModel when saving changes.